plugin/auth_ldap/tests/mtr/auth_ldap_simple.test

```diff
@@ -6,8 +6,8 @@
 SELECT PLUGIN_NAME, PLUGIN_STATUS FROM INFORMATION_SCHEMA.PLUGINS WHERE PLUGIN_NAME LIKE 'authentication_ldap_simple%';
 SHOW GLOBAL VARIABLES LIKE 'authentication_ldap_simple%';
 SET GLOBAL authentication_ldap_simple_bind_base_dn = 'ou=people,dc=planetexpress,dc=com';
-SET GLOBAL authentication_ldap_simple_server_host = '172.18.0.2';
-SET GLOBAL authentication_ldap_simple_server_port = 389;
+--eval SET GLOBAL authentication_ldap_simple_server_host = '$MTR_LDAP_HOST'
```

Comment:
Add `--replace-result $MTR_LDAP_HOST <MTR_LDAP_HOST>`
and 
`--replace-result $MTR_LDAP_PORT <MTR_LDAP_PORT>`
here